pkg/securitycontext/util_test.go

```diff
@@ -120,7 +121,7 @@ func TestHaRootUID(t *testing.T) {
 }
 
 func TestHasRunAsUser(t *testing.T) {
-	var runAsUser int64 = 0
+	runAsUser := types.UnixUserID(0)
```

Comment:
In the topic of making it easier to review, this makes it harder.  Trivial changes are easy to scan.  The requires reading.